### PR TITLE
Feature/implement zoom in and out controls

### PR DESCRIPTION
**Pull request by matbmapspeople**
For Mapbox: I added Mapbox zoom controls and for those controls I applied styling so SVGs can be used as well as it will align with Figma design.

For Google Maps: For now I just added working zoom controls.

For both: I added two SVGs.

### CODE REVIEW

File: packages/map-template/src/components/MapTemplate/MapTemplate.scss [hunk 1]
```diff
@@ -55,7 +85,7 @@
     position: relative;
 
     #mi_map_logo {
-        z-index: var(--z-index-dropdown) !important;
+        z-index: var(--z-index-dropdown);
```
Comment (ammapspeople) [inline packages/map-template/src/components/MapTemplate/MapTemplate.scss:88]: Why is this change necessary?

Comment (matbmapspeople) [reply packages/map-template/src/components/MapTemplate/MapTemplate.scss]: Removed by an accident. Reverted.

File: packages/map-template/src/components/MapTemplate/MapTemplate.scss [hunk 2]
```diff
@@ -47,6 +47,46 @@
     .mi-floor-selector__list {
         box-sizing: unset;
     }
+
+    .gmnoprint {
+        background-color: red;
```
Comment (ammapspeople) [inline packages/map-template/src/components/MapTemplate/MapTemplate.scss:52]: Is this intended?

image

Comment (matbmapspeople) [reply packages/map-template/src/components/MapTemplate/MapTemplate.scss]: Removed.

File: packages/map-template/src/App.jsx [hunk 3]
```diff
@@ -7,7 +7,7 @@ function App() {
             {/* This is the Map Template component */}
             <MapsIndoorsMap supportsUrlParameters={true}
                 gmApiKey={import.meta.env.VITE_GOOGLE_MAPS_API_KEY}
-                mapboxAccessToken={import.meta.env.VITE_MAPBOX_ACCESS_TOKEN} 
+                // mapboxAccessToken={import.meta.env.VITE_MAPBOX_ACCESS_TOKEN}
```
[the remark this answers is not in the document]
Comment (matbmapspeople) [reply packages/map-template/src/App.jsx]: Forgot about this due to testing Google Maps zoom controls. Uncommented.

File: packages/map-template/src/components/MapTemplate/MapTemplate.scss [hunk 4]
```diff
@@ -47,6 +47,44 @@
     .mi-floor-selector__list {
         box-sizing: unset;
     }
+
+    .gmnoprint {
```
Comment (ammapspeople) [inline packages/map-template/src/components/MapTemplate/MapTemplate.scss:51]: Can you elaborate in a code comment why we are doing this?

Comment (matbmapspeople) [reply packages/map-template/src/components/MapTemplate/MapTemplate.scss:51]: Done ✅ . I will show it to UX department soon for the final approval

Comment (ammapspeople) [reply packages/map-template/src/components/MapTemplate/MapTemplate.scss:51]: I don't see it 😄 

Also, can these map-specific styles be placed in a GoogleMapsMap.scss file and a MapboxMap.scss file in their respective component folders?

**Comment by matbmapspeople** [comment]
Ready for re-review @ammapspeople

File: packages/map-template/src/components/Map/GoogleMapsMap/GoogleMapsMap.scss [hunk 5]
```diff
@@ -0,0 +1,10 @@
+.mapsindoors-map {
```
Comment (ammapspeople) [inline packages/map-template/src/components/Map/GoogleMapsMap/GoogleMapsMap.scss:1]: Can we use another selector, one that is more specific to the GoogleMapsMap component?

**Comment by ammapspeople** [comment]
If you resolve the git conflicts, I will approve 😄

**Review by ammapspeople — APPROVED** [review]
👏

File: packages/map-template/src/components/Map/MapboxMap/MapboxMap.scss [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+            background-repeat: no-repeat;
+            background-size: 20px 20px;
+            background-position: center center;
+        }
+
+        &-icon {
+            background-image: none;
+        }
+    }
+}
```
Comment (andreeaceachir142) [inline packages/map-template/src/components/Map/MapboxMap/MapboxMap.scss:31]: For consistency reasons, could we use midt variables here? 🙏🏻
See examples in the other `scss` files @matbmapspeople
